MinimumFalling.py

```diff
@@ -0,0 +1,13 @@
+class Solution:
+    def minFallingPathSum(self, A: List[List[int]]) -> int:
+        for i in range(1,len(A)):
+            for j in range(len(A[0])):
+                if(i-1>0 and j+1>0 and j+1<len(A[i])and i-1<len(A[i])):
```

Comment:
if and elif conditions are wrong for the boundary conditions, check again. you should be able to get the correct answers